### ISSUE TITLE
'DOCUMENT' was not found in '@angular/platform-browser'

### ISSUE THREAD

**Issue by sGerli** [Teradata/covalent-text-editor-nightly#27]
In angular 8 `DOCUMENT` was moved from `@angular/platform-browser` to `@angular/common`.
https://angular.io/api/common/DOCUMENT
I changed that in my local copy and it seems to be working.

```
ERROR in ./node_modules/@covalent/text-editor/fesm5/covalent-text-editor.js 382:63-71
"export 'DOCUMENT' was not found in '@angular/platform-browser'
```

**Comment by zaalbarxx** [comment]
Also experienced this problem, created a pull request for it, could you take a look and eventually merge it and publish on npm ? #28

**Comment by jeremysmartt** [comment]
closed by #28
